openapi/2023-12.yaml

```diff
@@ -365,6 +428,35 @@ components:
                       type: string
                       example: ""
   schemas:
+    Flow:
+      type:
+        - object
+      required:
+        - type
+        - attributes
+      properties:
+        type:
+          type: string
+          example: "flows"
```

Comment:
Nit: not just here, but everywhere we use type, there is only on possible value. We could add an `enum: ['flows']` property.

Reply:
good idea